src/Compilers/VisualBasic/Test/Emit/Attributes/EmitTestStrongNameProvider.vb

```diff
@@ -1,18 +1,26 @@
-﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 Imports System.IO
+Imports System.Reflection.Metadata
+Imports System.Security.Cryptography
+Imports Microsoft.Cci
 Imports Microsoft.CodeAnalysis
 Imports Microsoft.CodeAnalysis.VisualBasic
 Imports Microsoft.CodeAnalysis.VisualBasic.UnitTests
+Imports Roslyn.Test.Utilities.SigningTestHelpers
 
 Partial Public Class InternalsVisibleToAndStrongNameTests
     Inherits BasicTestBase
 
+    Private Shared ReadOnly BYPASS_STRONGNAME_FEATURE As KeyValuePair(Of String, String)() = {New KeyValuePair(Of String, String)("BypassStrongName", "+")}
```

Comment:
Name does not match casing guidelines. But since this is only used in one location, consider inlining instead.

Reply:
Oh, I must have missed that one.  This was actually temporary.  I've removed it and its usage.